### DISCUSSION TITLE
Links to Legacy Update are being blocked on Twitter [UPDATE: now resolved]

### DISCUSSION THREAD

**Discussion by kirb** [LegacyUpdate/LegacyUpdate#56] [Announcements]
Just wanted to make a quick note that Legacy Update seems to have been caught up in the wide array of links being blocked by Twitter in the past two days.

I’m very disappointed in Twitter for this, but not surprised, I guess. It’s unclear what I did wrong — my initial assumption was that it got blocked because of the link to my Mastodon profile in the sidebar, being one of the first domains Twitter targeted with blocks. Despite hiding this link when Twitterbot visits the site, and waiting 12 hours, the block hasn’t been lifted.

I’m legitimately lost as to why this happened, and I’m sorry if the scary warnings from Twitter concern you. Unfortunately, I don’t think I’ll be hearing anything back from them as to what I can do to resolve this, since everyone I knew at Twitter has understandably left, and the current team have been tight-lipped about the blocking of Mastodon links (strange for how talkative the new CEO has otherwise been).

**To be clear, Legacy Update has not been compromised, and does not do anything other than what it clearly says it’s doing.** Here is the [VirusTotal report](https://www.virustotal.com/gui/url/367c2c48993800d420427e3022f4476f3d838434197ad12321d54751a5dfdde7) for the domain; only 4 out of 91 vendors are considering it malware. (I’ll be sending false-positive reports so they can hopefully correct this.)

**Comment by caraar12345** [comment]
On this note, NextDNS seems to be blocking `content.legacyupdate.net` as part of "AI-Driven Threat Detection". So possibly related!
![Screenshot_20221223_155018](https://user-images.githubusercontent.com/6129428/209372836-b3c05218-42be-4731-b60c-b7aae1f35c07.png)

**Reply by kirb** [reply]
Ugh. Thanks, I’ll try and figure out where to report false-positive to. The main legacyupdate.net domain is fine?

**Comment by kirb** [comment]
I forgot to update this thread. This was resolved a few weeks ago thanks to the vendors I reached out to, in addition to some users who themselves also reported false-positives. Thanks heaps to those people who helped get this sorted.

**Comment by TubaOrNotTuba** [comment]
Barracuda's content blocking system is also blocking legacyupdate.net.
![lu](https://github.com/kirb/LegacyUpdate/assets/27431511/d4d42219-6cc4-4b5f-85f9-e5cf1ac48ef5)
